Tools/ardupilotwaf/boards.py

```diff
@@ -44,7 +44,7 @@ def configure(self, cfg):
                     keys.sort()
                 val = ['%s=%s' % (vk, val[vk]) for vk in keys]
 
-            if k in cfg.env and isinstance(cfg.env, list):
+            if k in cfg.env and isinstance(cfg.env.k, list):
```

Comment:
I get what you are saying but I'm wondering how it actually worked... Because the Clang flags in C/CXXFLAGS were getting overwritten instead of merged and this solved it... How would ''CXXFLAGS'' be a list?
Anyway I will correct it.